src/components/ContinuationIn/AuthorizationProof.vue

```diff
@@ -334,6 +334,12 @@ export default class ExtraproRegistration extends Mixins(DocumentMixin) {
    * @param file the file to add
    */
   async onFileSelected (file: File): Promise<void> {
+    /** Returns True if given string contains only Latin 1 (ie, ISO8859-1) characters. */
+    function isValidLatin1 (str: string): boolean {
+      // eslint-disable-next-line no-control-regex
+      return !/[^\u0000-\u00ff]/g.test(str)
+    }
```

Comment:
I did a bit of research on how to identify filenames with invalid characters and this is the best / cleanest I could find. (Other solutions used libraries or packages and made other assumptions.)

This arose from a file that Janis was trying to upload and which caused an error while trying to upload the file.  Now the following error is displayed:

![image](https://github.com/user-attachments/assets/3c910899-33bb-4ab6-b7b6-11bcf1add186)

Sample file with invalid character in filename:
[Maybe it’s time (invalid filename).pdf](https://github.com/user-attachments/files/17215019/Maybe.it.s.time.invalid.filename.pdf)